src/client/Microsoft.Identity.Client/WsTrust/WsTrustWebRequestManager.cs

```diff
@@ -26,8 +28,10 @@ public WsTrustWebRequestManager(IHttpManager httpManager)
         /// <inheritdoc/>
         public async Task<MexDocument> GetMexDocumentAsync(string federationMetadataUrl, RequestContext requestContext)
         {
+            IDictionary<string, string> headers = CreateHttpTelemetryHeaders(requestContext);
+
             var uri = new UriBuilder(federationMetadataUrl);
-            HttpResponse httpResponse = await _httpManager.SendGetAsync(uri.Uri, null, requestContext.Logger).ConfigureAwait(false);
+            HttpResponse httpResponse = await _httpManager.SendGetAsync(uri.Uri, headers, requestContext.Logger).ConfigureAwait(false);
```

Comment:
I don't think the Ws Trust endpoint supports this, nor do I think they will ever support this, so might as well not send the data.

Reply:
That said, we should definitely send the MsalIDParameters here.